gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/task/AbstractJKubeTask.java

```diff
@@ -76,6 +76,7 @@ public final void runTask() {
 
   private void init() {
     kubernetesExtension.javaProject = GradleUtil.convertGradleProject(getProject());
+    kubernetesExtension.javaProject.setPlugins(GradleUtil.getPluginWithKubernetesExtensionConfigured(kubernetesExtension.javaProject.getPlugins(), kubernetesExtension.javaProject.getGradlePlugins(), kubernetesExtension));
```

Comment:
Same as the previous comments, we're setting again the javaProject plugins which were initially instantiated in the `GradleUtil.convertGradleProject`.

Ideally the `JavaProject` class should be immutable, we might want to evaluate if this is possible.